components/alertbanner/index.css

```diff
@@ -0,0 +1,128 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-AlertBanner {
+  --spectrum-alert-banner-min-height: var(--spectrum-alert-banner-minimum-height);
+  --spectrum-alert-banner-size: var(--spectrum-alert-banner-width);
+  --spectrum-alert-banner-font-size: var(--spectrum-font-size-100);
+  --spectrum-alert-banner-icon-size: var(--spectrum-workflow-icon-size-100);
+
+  /* spacing */
+  --spectrum-alert-banner-icon-to-text: var(--spectrum-text-to-visual-300);
+  --spectrum-alert-banner-start-edge: var(--spectrum-spacing-300);
+  --spectrum-alert-banner-text-to-button-horizontal: var(--spectrum-spacing-300);
+  --spectrum-alert-banner-text-to-divider: var(--spectrum-spacing-300);
+  --spectrum-alert-banner-top-icon: var(--spectrum-alert-banner-top-to-workflow-icon);
+  --spectrum-alert-banner-top-text: var(--spectrum-alert-banner-top-to-text);
+  --spectrum-alert-banner-bottom-text: var(--spectrum-alert-banner-bottom-to-text);
+  --spectrum-alert-banner-close-button-spacing: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-edge-to-divider: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-edge-to-button: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-text-to-button-vertical: var(--spectrum-spacing-100);
+
+  /* colors */
+  --spectrum-alert-banner-informative-background: var(--spectrum-informative-background-color-default);
+  --spectrum-alert-banner-negative-background: var(--spectrum-negative-background-color-default);
+  --spectrum-alert-banner-font-color: var(--spectrum-white);
+
+  .spectrum--large & {
+    --spectrum-alert-banner-close-button-spacing: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-edge-to-divider: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-edge-to-button: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-text-to-button-vertical: var(--spectrum-spacing-200);
+  }
```

Comment:
We'll need to pull these out and place them into `tokens/custom-spectrum/custom-large-vars.css`, as SWC isn't currently set up to take in these scale-related custom properties from the components themselves.

Reply:
got this changed, are the custom props good to stay with in this PR or do they need to be moved into a separate one?